recipes/cpprestsdk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        if self.settings.compiler == 'Visual Studio':
+            with tools.vcvars(self.settings, force=True, filter_known_paths=False):
+                self.build_cmake()
+        else:
+            self.build_cmake()
+
+    def build_cmake(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
```

Comment:
Correct me if I'm wrong or missing anything, but `CMake` does not need `vcvars`.
```suggestion
    def build(self):
        if self.settings.compiler == 'Visual Studio':
            cmake = CMake(self)
            cmake.configure()
            cmake.build()
```

Reply:
you are correct, tested locally